build.gradle

```diff
@@ -33,6 +33,15 @@ allprojects {
         jcenter()
         maven { url 'https://jitpack.io' }
     }
+
+    gradle.projectsEvaluated {
+        tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+            kotlinOptions {
+                apiVersion = '1.3'
+                languageVersion = '1.3'
```

Comment:
Ok, nice way to block 1.4 but this should be not forever.
Please can you tell me, what blocks you, to have a better understanding.
Are lint rules a blocker ?

Reply:
I think it could be best practice to support one version behind for all lib development if possible.  I don't mind for example when you change to Kotlin 1.5.xx, by then most if not all lib users should be using `1.4`, and then you can start using `1.4` language features in the lib itself that won't be deprecated in `1.5`

I recommend to change `1.3` to `1.4` for example when you move to Kotlin 1.5.xx, the binary format is mostly forwards compatible with the next feature release, but not later ones (in the cases when new features are not used, e.g. 1.4 can understand most binaries from 1.5, but not 1.6)

Even without this code block, the lib in its current state will run with Kotlin 1.3.xx only project

My personal project, well I have decided to lock Kotlin to 1.3.72, nothing blocks me, but I have yet to migrate all deprecated code to finally make the move to Kotlin 1.4.xx, the block is imposed by me in this case...